app/controllers/v1/mixtapes_controller.rb

```diff
@@ -0,0 +1,14 @@
+class V1::MixtapesController < V1::ApplicationController
+
+  def index
+
+  end
+
+  def create()
+    mixtape = MixtapeMaker.generate(params[:no_song], artists: params[:artist])
```

Comment:
Often when we see params/argument that match models, our first instinct is to look at these as a collection of `Artist` ... but in this case it isn't, but that confused me.

Do you want it to be a list of `artist_names` or just one `artist_name`?